specification/orbital/resource-manager/Microsoft.Orbital/aodg/preview/2024-03-01-preview/aodg.json

```diff
@@ -0,0 +1,1750 @@
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The Azure Resource ID."
+              }
+            }
+          },
+          "description": "A list of Azure Resource IDs."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for an API service call that lists the resource IDs of resources associated with another resource."
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Creating",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Updating",
+        "Deleting"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      },
+      "description": "The current state of the resource's creation, deletion, or modification.",
+      "readOnly": true
+    },
+    "GroundStationsProperties": {
+      "type": "object",
+      "required": [
+        "capabilities",
+        "globalCommunicationsSite"
+      ],
+      "properties": {
+        "city": {
+          "type": "string",
+          "description": "City of ground station."
+        },
+        "capabilities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Capability"
+          },
+          "description": "Ground station capabilities."
+        },
+        "providerName": {
+          "type": "string",
+          "description": "Ground station provider name."
+        },
+        "longitudeDegrees": {
+          "type": "number",
+          "description": "Longitude of the ground station in decimal degrees."
+        },
+        "latitudeDegrees": {
+          "type": "number",
+          "description": "Latitude of the ground station in decimal degrees."
+        },
+        "altitudeMeters": {
+          "type": "number",
+          "description": "Altitude of the ground station."
+        },
+        "releaseMode": {
+          "type": "string",
+          "enum": [
+            "Preview",
+            "GA"
+          ],
+          "x-ms-enum": {
+            "name": "releaseMode",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "Release Status of a ground station."
+        },
+        "globalCommunicationsSite": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "required": [
+            "id"
+          ],
+          "description": "A reference to Azure Orbital Digital Ground site.",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "The Azure Orbital Digital Ground site's resource ID."
```

Comment:
The description doesn't seem according to the property name here.

Reply:
Updated